Cargo.toml

```diff
@@ -1,20 +1,73 @@
 [package]
-
-name         = "proteus"
-version      = "1.0.4"
+name         = "proteus-wasm"
+version      = "2.1.0"
 authors      = ["Wire Swiss GmbH <seceng@wire.com>"]
 description  = "Axolotl protocol implementation"
 homepage     = "https://github.com/wireapp/proteus"
-repository   = "git@github.com:wireapp/proteus.git"
+repository   = "https://github.com/wireapp/proteus.git"
 license      = "GPL-3.0"
+edition      = "2021"
+
+[workspace]
+members = [
+    "crates/proteus-traits",
+    "crates/cbor-codec",
+]
+
+[patch.'https://github.com/wireapp/proteus'.proteus]
+package = "proteus"
+# Keep the extra / in the url, it prevents this error:
+# `patch for `proteus` in `https://github.com/wireapp/proteus` points to the same source, but patches must point to different sources`
+git = "https://github.com/wireapp//proteus"
+branch = "otak/fix-1.0.3"
```

Comment:
Is this intended? if yes, doesn't this defeat the purpose of having a single (branch) source of truth?

Reply:
This is to test with older version to ensure it is retro-compatible